secboot/secboot_sb.go

```diff
@@ -171,7 +186,12 @@ func UnlockVolumeUsingSealedKeyIfEncrypted(disk disks.Disk, name string, sealedE
 
 	sbSetBootMode(opts.BootMode)
 	defer sbSetBootMode("")
-	sbSetKeyRevealer(&keyRevealerV3{})
+
+	if fdeHookPresent {
```

Comment:
We can only have one key revealer?

I wonder if we should have maybe encoded the type of key in the handle. And have only one revealer that decides what to do.

That is the data should be json map with a special field to tell what type. If it does not have the special field, then we fallback to old format.